pkg/cmd/server/kubernetes/master.go

```diff
@@ -97,6 +105,22 @@ func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string
 	recycler.Run()
 }
 
+// attemptToLoadRecycler tries decoding a pod from a filepath for use as a recycler for a volume.
+// If successful, this method will set the recycler on the config.
```

Comment:
Comment is incorrect (see below), this method can return success without setting a pod template